trace-dispatcher/src/Cardano/Logging/Forwarding.hs

```diff
@@ -307,8 +308,9 @@ doListenToAcceptor magic snocket makeBearer configureSocket address timeLimits
  where
   forwarderApp
     :: [(RunMiniProtocol 'Mux.ResponderMode initiatorCtx responderCtx LBS.ByteString IO Void (), Word16)]
+    -> ForwardingVersionData
     -> OuroborosApplication 'Mux.ResponderMode initiatorCtx responderCtx LBS.ByteString IO Void ()
-  forwarderApp protocols =
+  forwarderApp protocols _ =
```

Comment:
As before, there's no motivation to change type signature of `forwarderApp` (twice). Can that be reverted?